apps/ashley/templates/services/nginx/configs/healthcheck.conf.j2

```diff
@@ -0,0 +1,9 @@
+server {
+  listen {{ ashley_nginx_healthcheck_port }};
+  server_name localhost;
+
+  location {{ ashley_nginx_healthcheck_endpoint }} {
+    add_header Content-Type text/plain;
```

Comment:
I've added the following for `etherpad` to not pollute logs, WDYT?

```suggestion
    add_header Content-Type text/plain;
    access_log off;
```

If you agree, this should be backported for other apps IMO (in a new PR).

Reply:
I agree with you,  good suggestion ! :+1: